string ready_to_pick
---
ball_launch_service_answer answer